views/templates/block/activityLog.html.twig

```diff
@@ -1,23 +1,23 @@
 <div class="bootstrap" id="activityLogBlock" style="display:none">
     <div class="panel">
         <div class="panel-heading">
-            {{ 'Activity Log'|trans }}
+            {{ 'Activity Log'|trans({}, 'Modules.Autoupgrade.Admin') }}
```

Comment:
Note that it's useless to add translation domain, this module uses its own translator, which goes straight to the legacy translation system.

See:
- https://github.com/PrestaShop/autoupgrade/blob/v4.13.0/classes/UpgradeTools/Translator.php
- https://github.com/PrestaShop/autoupgrade/blob/v4.13.0/classes/Twig/TransFilterExtension.php

The wordings must be added to the legacy dictionaries as explained in the docs

Reply:
What we could do is reimplement the translator within the module to use xliff files. That way the dictionaries could be created using the BO interface in 1.7.8, then reused everywhere. And it also would allow to break one dependency from the core.